feature/experimental/p4rt/internal/p4rtutils/p4rtutils.go

```diff
@@ -259,6 +259,9 @@ func P4RTNodesByPort(t testing.TB, dut *ondatra.DUTDevice) map[string]string {
 		case ondatra.CISCO:
 			return inferP4RTNodesCisco(t, dut)
 		case ondatra.NOKIA:
+			if *args.P4RTNodeName1 != "" && *args.P4RTNodeName2 != "" {
```

Comment:
Suggest moving this to the `inferP4RTNodesNokia()` function.

Reply:
Done, thx